scripts/genclient.sh

```diff
@@ -16,7 +15,8 @@ readonly DEVICES_DIR=${SERVER_DIR}/devices
 readonly DIR=${DEVICES_DIR}/${DEVICE_ID}
 mkdir -p ${DIR}
 
-openssl ecparam -genkey -name prime256v1 | openssl ec -out ${DIR}/pkey.pem
+openssl ecparam -genkey -name prime256v1 | openssl ec -out ${DIR}/pkey.ec.pem
+openssl pkcs8 -topk8 -nocrypt -in ${DIR}/pkey.ec.pem -out ${DIR}/pkey.pem
```

Comment:
I needed to convert to pkcs8 to get the `softhsm` working on qemu. Can you confirm if this is really needed @patrickvacek ?

Reply:
I'll try to run through it tomorrow. I've never had to do that before, but perhaps the way you're setting things up is a bit different than what I've done before.

Reply:
I'm having trouble trying to test this myself because I can't tell where the client and CA certs are coming from. It looks like the first private key you generate is an EC key, which currently I don't think aktualizr entirely supports. (It should, but I think there are a few things missing/broken at present.) So your conversion might only be necessary to get an RSA key. Or at least, that's what it looks like is happening.